Cargo.toml

```diff
@@ -18,6 +18,11 @@ gimli={git="https://github.com/fitzgen/gimli.git"}
 elf = "0.0.9"
 log = "0.3.6"
 env_logger = "0.3.4"
+mach = "0.0.5"
+libproc = "0.1.0"
+object = "0.1.0"
+libarchive = "0.1.1"
+libarchive3-sys = "0.1.2"
```

Comment:
could you move all the deps you added to a `[target.'cfg(target_os="macos")'.dependencies]` section? It would stop requiring libarchive on Linux where it's not needed, or at least not yet needed.
`